frontend/src/views/partner/AddPartnerbyURL.vue

```diff
@@ -206,6 +216,29 @@ export default {
           EventBus.$emit("error", this.$axiosErrorMessage(error));
         });
     },
+    copyInvitationURL() {
+      let idElement = document.querySelector(
```

Comment:
This does not look like the vue js way and I think the better way would be to use `this.$refs.clone.focus();` like it is done here: https://visualbean.medium.com/how-to-copy-to-clipboard-with-vuejs-a1b7ffb1170e to get the reference to the text box.

Reply:
Thanks for the link, also see this https://github.com/hyperledger-labs/business-partner-agent/pull/752
which makes it consistent in schema management.